test/e2e/samples_test.go

```diff
@@ -69,6 +71,7 @@ func createBuilders(t *testing.T, decoder *helper.YAMLDecoder, sampleFile, testN
 				WithElasticsearchRef(tweakServiceRef(b.ApmServer.Spec.ElasticsearchRef, suffix)).
 				WithKibanaRef(tweakServiceRef(b.ApmServer.Spec.KibanaRef, suffix)).
 				WithConfig(map[string]interface{}{"apm-server.ilm.enabled": false}).
+				WithoutIntegrationCheck().
```

Comment:
e2e tests were failing for APM without this on 8.x because it's required... maybe I'll move this to another PR.  this is probably/likely replicated in #5470 .

Reply:
Yeah, it's right here https://github.com/elastic/cloud-on-k8s/pull/5470/files#diff-74cf70d92121efd35da5334a5e656631c6f194141a391df7f9c99456b8d35cbeR76
removing from here.